js/account-popover.jquery.js

```diff
@@ -118,6 +121,25 @@
             $trigger.popover('hide');
           }
         });
+
+        // Smaller screens get the old popover
+        $(window).on('id7:reflow', $.proxy(function (e, screenConfig) {
+          this.options.useMwIframe = !(screenConfig.name === 'xs')
```

Comment:
Not a massive deal but I'd rather we treated `options` as immutable and had `useMwIframe` as an instance variable on the instance of the plugin